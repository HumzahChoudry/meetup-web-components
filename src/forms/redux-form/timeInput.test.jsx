import React from 'react';
import { shallow } from 'enzyme';
import ReduxFormTimeInput from './TimeInput';

describe('redux-form TimeInput', function() {

	const formAttrs = {
		input: {
			label: 'What time is it?',
			name: 'partytime',
			value: '22:00',
			required: true
		},
		meta: {
			error: 'Now approaching midnight!!?'
		}
	};

	it('renders a TimeInput component with expected attributes from mock data', () => {
		const component = shallow(<ReduxFormTimeInput {...formAttrs} />);

		expect(component).toMatchSnapshot();
	});
});
