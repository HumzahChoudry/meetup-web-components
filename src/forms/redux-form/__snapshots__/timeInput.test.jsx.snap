// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`redux-form TimeInput renders a TextInput component with expected attributes from mock data 1`] = `
<TimeInput
  error="Now approaching midnight!!?"
  label="What time is it?"
  name="partytime"
  required={true}
  value="22:00"
/>
`;
